pkg/operator/provisioner/provisioner.go

```diff
@@ -127,7 +130,7 @@ func (p *RookVolumeProvisioner) Provision(options controller.VolumeOptions) (*v1
 					CephMonitors: monitors,
 					RadosUser:    radosUser,
 					SecretRef:    secretRef,
-					FSType:       "ext4",
+					FSType:       p.provConfig.fstype,
```

Comment:
Here. https://github.com/kubernetes/kubernetes/blob/master/pkg/util/mount/mount_linux.go#L369

All the plugin eventually call this function for mounting and formatting.